CONTRIBUTING.md

```diff
@@ -76,3 +78,4 @@ curve, so keep it up!
 [verification steps]:https://help.github.com/articles/writing-on-github#task-lists
 [reference associated issues]:https://github.com/blog/1506-closing-issues-via-pull-requests
 [Issues]:https://github.com/rapid7/insightconnect-plugins/issues
+[developer mailing list]:https://groups.google.com/a/rapid7.com/d/forum/rapid7-insightconnect-development
```

Comment:
Why aren't we using proper capitalization in half of these links?
`Developer mailing list`